### DISCUSSION TITLE
Stitches Abandonware?

### DISCUSSION THREAD

**Discussion by datner** [heroui-inc/heroui#755] [Help] [answered]
Hey, looking at the state of stitches and how it's been going the last few months (at least from where I'm looking), 
and it seems to be in the process of being abandoned..

Is this noticed? Are there plans around this subject?
How does nextui plan to operate if stitches is indeed abandoned?

**Comment by jrgarciadev** [comment, marked as answer]
Hey @datner, in case of Stitches is abandoned we planned to create a built-in styled solution based on top of Stitches, but according to the new owners of Modulz they will continue supporting Stitches https://twitter.com/zenorocha/status/1532007032831631360

**Reply by datner** [reply]
Seems like they placed it on Life Support though, and not actively developing.. 
Kinda frustrated to see them willing to continue with other libraries adjacent to stitches but not stitches itself.
Though it is understandable, since it's a very complicated project and is probably very hard to iterate on
